### PR TITLE
add the `enable-disk-spill-encrypt.md` to descript the usage of the `spilled-file-encryption-method`

### PR DESCRIPTION
**Pull request by mmyj**
### What is changed, added or deleted? (Required)

add the `enable-disk-spill-encrypt.md` to descript the usage of the `spilled-file-encryption-method`.

### Which TiDB version(s) do your changes apply to? (Required)

- [X] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/tidb/pull/19070
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/4302

### CODE REVIEW

File: enable-disk-spill-encrypt.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: 为 TiDB 落盘文件开启加密
+aliases: ['/docs-cn/dev/enable-tls-between-clients-and-servers/','/docs-cn/dev/how-to/secure/enable-tls-clients/','/docs-cn/dev/encrypted-connections-with-tls-protocols/','/docs-cn/dev/enable-tls-between-clients/']
+---
+
+# 为 TiDB 落盘文件开启加密
+
+当配置项 `oom-use-tmp-storage` 为 `ture` 时，如果单条 SQL 语句的内存使用超出 `mem-quota-query` 的限制，某些算子可以将执行时的中间结果作为临时文件落盘保存，直到算子执行完成之后将它们删除。可以开启落盘文件加密功能，防止攻击者通过读取临时文件来访问数据。
```
Comment (XuHuaiyu) [inline enable-disk-spill-encrypt.md:7]: ture --> true

Comment (mmyj) [reply enable-disk-spill-encrypt.md]: fixed

File: enable-disk-spill-encrypt.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: 为 TiDB 落盘文件开启加密
```
Comment (XuHuaiyu) [inline enable-disk-spill-encrypt.md:2]: @TomShawn Hi, can you help to check what's the alias of this file?

File: enable-disk-spill-encrypt.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+# 为 TiDB 落盘文件开启加密
+
+当配置项 `oom-use-tmp-storage` 为 `true` 时，如果单条 SQL 语句的内存使用超出 `mem-quota-query` 的限制，某些算子可以将执行时的中间结果作为临时文件落盘保存，直到算子执行完成之后将它们删除。可以开启落盘文件加密功能，防止攻击者通过读取临时文件来访问数据。
```
Comment (XuHuaiyu) [inline enable-disk-spill-encrypt.md:7]: 算子执行完成 --> 查询执行完成

File: enable-disk-spill-encrypt.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## 配置加密
+
+要启用落盘文件加密功能，可以在 TiDB 的配置文件中添加安全相关配置
+
+```
+[security]
+spilled-file-encryption-method = aes128-ctr
```
Comment (XuHuaiyu) [inline enable-disk-spill-encrypt.md:15]: "aes128-ctr" ?

File: enable-disk-spill-encrypt.md (hunk 3 above)
Comment (mmyj) [reply enable-disk-spill-encrypt.md]: fixed

File: enable-disk-spill-encrypt.md (hunk 4 above)
Comment (mmyj) [reply enable-disk-spill-encrypt.md]: fixed

**Comment by XuHuaiyu** [comment]
LGTM 

PTAL @SunRunAway @TomShawn

File: enable-disk-spill-encrypt.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+# 为 TiDB 落盘文件开启加密
+
+当配置项 `oom-use-tmp-storage` 为 `true` 时，如果单条 SQL 语句的内存使用超出 `mem-quota-query` 的限制，某些算子可以将执行时的中间结果作为临时文件落盘保存，直到查询执行完成之后将它们删除。可以开启落盘文件加密功能，防止攻击者通过读取临时文件来访问数据。
+
+## 配置加密
+
+要启用落盘文件加密功能，可以在 TiDB 的配置文件中添加安全相关配置
+
+```
+[security]
```
Comment (SunRunAway) [inline enable-disk-spill-encrypt.md:14]: 这样描述会不会有用户无脑把这一篇粘贴到配置文件中。
是不是描述成 
> 在 TiDB 配置文件中的 `[security]` 部分，设置 `spilled-file-encryption-method` 选项。

Comment (mmyj) [reply enable-disk-spill-encrypt.md:14]: 需要加上一句

Comment (mmyj) [reply enable-disk-spill-encrypt.md:14]: 用户如果发现没有`[security]`，会知道加一个的吗？需要再详细指明吗？

Comment (SunRunAway) [reply enable-disk-spill-encrypt.md:14]: 直接指个链接到这页相应的配置上好了。https://docs.pingcap.com/zh/tidb/stable/tidb-configuration-file

Comment (mmyj) [reply enable-disk-spill-encrypt.md:14]: ok

Comment (mmyj) [reply enable-disk-spill-encrypt.md:14]: fixed.

**Comment by mmyj** [comment]
PTAL @SunRunAway @TomShawn

File: enable-disk-spill-encrypt.md (hunk 2 above)
Comment (TomShawn) [reply enable-disk-spill-encrypt.md:2]: This is a new document, so it does not need an alias.

File: enable-disk-spill-encrypt.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+## 配置加密
+
+要启用落盘文件加密功能，可以在 TiDB 配置文件中的 `[security]` 部分，配置 [spilled-file-encryption-method](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file#spilled-file-encryption-method) 选项：
+
+```
+[security]
+spilled-file-encryption-method = "aes128-ctr"
+```
+
+`spilled-file-encryption-method` 的可选值为 `aes128-ctr`、`plaintext`。默认值为 `plaintext`，表示不启用加密。
```
Comment (TomShawn) [inline enable-disk-spill-encrypt.md:18]:
```suggestion
`spilled-file-encryption-method` 的可选值为 `aes128-ctr` 和 `plaintext`。默认值为 `plaintext`，表示不启用加密。
```

File: enable-disk-spill-encrypt.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+title: 为 TiDB 落盘文件开启加密
+---
+
+# 为 TiDB 落盘文件开启加密
+
+当配置项 `oom-use-tmp-storage` 为 `true` 时，如果单条 SQL 语句的内存使用超出 `mem-quota-query` 的限制，某些算子可以将执行时的中间结果作为临时文件落盘保存，直到查询执行完成之后将它们删除。可以开启落盘文件加密功能，防止攻击者通过读取临时文件来访问数据。
+
+## 配置加密
+
+要启用落盘文件加密功能，可以在 TiDB 配置文件中的 `[security]` 部分，配置 [spilled-file-encryption-method](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file#spilled-file-encryption-method) 选项：
```
Comment (TomShawn) [inline enable-disk-spill-encrypt.md:11]:
```suggestion
要启用落盘文件加密功能，可以在 TiDB 配置文件中的 `[security]` 部分，配置 [`spilled-file-encryption-method`](/tidb-configuration-file.md#spilled-file-encryption-method) 选项：
```

File: enable-disk-spill-encrypt.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+---
+title: 为 TiDB 落盘文件开启加密
+---
+
+# 为 TiDB 落盘文件开启加密
+
+当配置项 `oom-use-tmp-storage` 为 `true` 时，如果单条 SQL 语句的内存使用超出 `mem-quota-query` 的限制，某些算子可以将执行时的中间结果作为临时文件落盘保存，直到查询执行完成之后将它们删除。可以开启落盘文件加密功能，防止攻击者通过读取临时文件来访问数据。
```
Comment (TomShawn) [inline enable-disk-spill-encrypt.md:7]:
```suggestion
当配置项 `oom-use-tmp-storage` 为 `true` 时，如果单条 SQL 语句的内存使用超出 `mem-quota-query` 的限制，某些算子可以将执行时的中间结果作为临时文件落盘保存，直到查询执行完成之后将它们删除。

用户可以开启落盘文件加密功能，防止攻击者通过读取临时文件来访问数据。
```

File: enable-disk-spill-encrypt.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+## 配置加密
```
Comment (TomShawn) [inline enable-disk-spill-encrypt.md:9]:
```suggestion
## 配置方法
```

**Comment by TomShawn** [comment]
LGTM

File: enable-disk-spill-encrypt.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+---
+title: 为 TiDB 落盘文件开启加密
+---
```
Comment (TomShawn) [inline enable-disk-spill-encrypt.md:3]:
```suggestion
summary: 了解如何为 TiDB 落盘文件开启加密。
---
```

File: enable-disk-spill-encrypt.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+当配置项 `oom-use-tmp-storage` 为 `true` 时，如果单条 SQL 语句的内存使用超出 `mem-quota-query` 的限制，某些算子可以将执行时的中间结果作为临时文件落盘保存，直到查询执行完成之后将它们删除。
+
+用户可以开启落盘文件加密功能，防止攻击者通过读取临时文件来访问数据。
+
+## 配置方法
+
+要启用落盘文件加密功能，可以在 TiDB 配置文件中的 `[security]` 部分，配置 [`spilled-file-encryption-method`](/tidb-configuration-file.md#spilled-file-encryption-method) 选项：
+
+```
```
Comment (TomShawn) [inline enable-disk-spill-encrypt.md:16]:
```suggestion
```toml
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**